go/chat/sender.go

```diff
@@ -107,6 +107,42 @@ func (s *BlockingSender) addPrevPointersToMessage(ctx context.Context, msg chat1
 	return updated, nil
 }
 
+func (s *BlockingSender) getAllDeletedEdits(ctx context.Context, msg chat1.MessagePlaintext,
+	convID chat1.ConversationID) (chat1.MessagePlaintext, error) {
+
+	// Make sure this is a valid delete message
+	if msg.ClientHeader.MessageType != chat1.MessageType_DELETE {
+		return msg, nil
+	}
+	if msg.ClientHeader.Supersedes == 0 {
+		return msg, fmt.Errorf("getAllDeletedEdits: no supersedes specified")
+	}
```

Comment:
Should we assert that the `MessageBody` is empty? We're about to stomp on it, and it would be really confusing if the caller had put something in there expecting to keep it.